elementary/monitor/cli.py

```diff
@@ -46,7 +47,13 @@ def common_options(func):
         type=str,
         default=Config.DEFAULT_PROFILES_DIR,
         help="Specify your profiles dir where a profiles.yml is located, this could be a dbt profiles dir "
-             "(if your profiles dir is ~/.dbt, no need to provide this parameter as we use it as default).",
+             "(if your profiles dir is ~/.dbt, no need to provide this parameter as we use it as default)."
+    )(func)
+    func = click.option(
+        '--target-dir', '-td',
```

Comment:
Maybe let's use `--target-path` to be consistent with the flag in config.yml?

Reply:
Agree, done